October212019.js

```diff
@@ -0,0 +1,46 @@
+let hacker1 = "Dan"
+console.log("The drivers name is "+ hacker1)
+let hacker2 ="The coldest that ever was"
+console.log("The navigator's name is "+hacker2)
+
+if(hacker1.length==hacker2.length){
+    console.log("Wow, you both got equally long names, "+hacker1.length+" characters!")
+}
+
+else if(hacker1.length >hacker2.length){
+    console.log("The Driver has the longest name, it has "+ hacker1.length+  " characters")
+}
+else{
+    console.log("Yo, navigator got the longest name, it has "+hacker2.length+" characters")
+}
+
+function nameSpace(name){
+let capitalName = name.toUpperCase()
+let spaceName=""
+for(i=0;i<name.length;i++){
+spaceName +=(capitalName[i] + " ")
+
+}
+return spaceName
+
+}
+
+function backwardsName(name){
+let backward=""
+for(i=0;i<=name.length-1;i++){
```

Comment:
I like this solution! The array.reverse() method is a great way of going about this as well.